src/DynamoApplications/PathResolvers.cs

```diff
@@ -30,6 +30,7 @@ public SandboxPathResolver(string preloaderLocation)
                 "DSCoreNodes.dll",
                 "DSOffice.dll",
                 "DSIronPython.dll",
+                "DSCPython.dll",
```

Comment:
I guess this should also be added to the CLI path resolver?

Reply:
Thanks @mjkkirschner , I will update that. Let me know if you find anything else.